File: content/atlas/source/includes/fts/tutorials/synonyms/CreateIndexMultiple.cs
using MongoDB.Bson;
using MongoDB.Driver;

// connect to your Atlas deployment
var uri = "<connection-string>";

var client = new MongoClient(uri);

var db = client.GetDatabase("sample_mflix");
var collection = db.GetCollection<BsonDocument>("movies");

// define your Atlas Search index
var index = new CreateSearchIndexModel(
  "default", new BsonDocument
  {
    { "mappings", new BsonDocument
      {
        { "dynamic", false },
        { "fields", new BsonDocument
          {
            { "title", new BsonDocument
              {
                { "analyzer", "lucene.english" },
                { "type", "string" }
              }
            }
          }
        }
      }
    },
    { "synonyms", new BsonArray
      {
        new BsonDocument
        {
          { "analyzer", "lucene.english" },
          { "name", "transportSynonyms" },
          { "source", new BsonDocument
            {
              { "collection", "transport_synonyms" }
            }
          }
        },
        new BsonDocument
        {
          { "analyzer", "lucene.english" },
          { "name", "attireSynonyms" },
          { "source", new BsonDocument
            {
              { "collection", "attire_synonyms" }
            }
          }
        }
      }
    }
  });

var result = collection.SearchIndexes.CreateOne(index);
Console.WriteLine($"New index name: {result}");

File: content/atlas/source/includes/fts/tutorials/synonyms/create-index-multiple.js
const { MongoClient } = require("mongodb");

// connect to your Atlas deployment
const uri =  "<connection-string>";

const client = new MongoClient(uri);

async function run() {
  try {

    // set namespace
    const database = client.db("sample_mflix");
    const collection = database.collection("movies");

    // define your Atlas Search index
    const index = {
        name: "default",
        definition: {
            "mappings": {
                "dynamic": false,
                "fields": {
                    "title": {
                        "analyzer": "lucene.english",
                        "type": "string"
                    }
                }
            },
            "synonyms": [
                {
                    "analyzer": "lucene.english",
                    "name": "transportSynonyms",
                    "source": {
                        "collection": "transport_synonyms"
                    }
                },
                {
                    "analyzer": "lucene.english",
                    "name": "attireSynonyms",
                    "source": {
                        "collection": "attire_synonyms"
                    }
                }
            ]
        }
    }

    // run the helper method
    const result = await collection.createSearchIndex(index);
    console.log("New index name: " + result);
  } finally {
    await client.close();
  }
}

run().catch(console.dir);